docs/resources.md

```diff
@@ -359,9 +358,49 @@ service account.
          secretName: bucket-sa
          secretKey: service_account.json
    ```
-
 ---
 
+#### BuildGCS Storage Resource
+BuildGCS storage resource points to
+[Google Cloud Storage](https://cloud.google.com/storage/) blob like  [GCS Storage Resource](#gcs-storage-resource) either in the form of a .zip archive, or based on the contents of a source manifest file.
+In addition to fetching an .zip archive, BuildGCS also unzips it.
+A [Source Manifest File](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/gcs-fetcher#source-manifests) is a JSON object listing other objects in Cloud Storage that should be fetched. The format of the manifest is a mapping of destination file path to the location in Cloud Storage where the file's contents can be found.
+BuildGCS resource can also do incremental uploads of sources via Source Manifest File.
+
+To create a BuildGCS type of storage resource using the `PipelineResource` CRD:
+
+```yaml
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: build-gcs-storage
+  namespace: default
+spec:
+  type: storage
+  params:
+    - name: type
+      value: build-gcs
+    - name: location
+      value: gs://build-crd-tests/rules_docker-master.zip
+    - name: artifactType
+      value: Archive
+```
+
+Params that can be added are the following:
+
+1. `location`: represents the location of the blob storage.
+2. `type`: represents the type of blob storage. For BuildGCS, this value should be set to `build-gcs`
+3. `artifactType`: represent the type of GCS resource. Right now, we support following types
+   - `Archive`:
+     - Archive indicates that resource fetched is an archive file. Currently, BuildGCS storage resource only supports `.zip` archive.
+     - It unzips the archive and places all the files in the directory which is set at runtime.
+     - If `artifactType` is set to `Archive`, `location` should point to a `.zip` file.
+   - `Manifest`:
+     - Manifest indicates that resource should be fetched using a source manifest file.
+     - If `artifactType` is set to `Manifest`, `location` should point to a source manifest file.
+
+Private buckets can not be configured as storage resources for BuildGCS Storage Resource right now.
```

Comment:
hm, is this true? I think you can still use the TaskRun's service account, or is something else required? (is it b/c gcs-fetcher doesn't recognize the service account credentials? if so it would help to explain that here in case we forget!)

Reply:
we can use the TaskRun's service account but nothing other than those. I will add that clause.